ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+## Investments
+Ubiquity's ability to retain the amount of investments received in the previous round commands deep respect. Such a situation is a strong positive signal to investors that their funds are managed efficiently. It is desirable for such information to be confirmed by facts that investors can verify.
+
+### Funding Round
+Since Ubiquity has funds from the previous round, it remains unclear for what purposes to attract a new investment round. To justify the need for additional funding, it is necessary to provide a reasoned development strategy for Ubiquity with directions for using capital.
+
+Typically, more effective times for investments are autumn and spring, summer being the least active period. It is better to start contacting investors at the beginning of autumn 2024.
+
+### Investors
+Experts asked whether Ubiquity is approaching current investors with an offer to participate in a new funding round. If investors familiar with Ubiquity are expanding their investments, this is a positive signal. If none of the existing investors wish to expand their investments in Ubiquity, this could be a negative signal. Investors communicate with each other and will try to get feedback about Ubiquity from current investors.
+
+### Valuation
+One of the approaches to determining valuation is market references. Experts listed several companies with which they will compare Ubiquity to determine valuation:
+
+- Tea protocol
+
+[https://tea.xyz/](https://tea.xyz/)
+
+- Talent protocol
+
+[https://www.talentprotocol.com/](https://www.talentprotocol.com/)
+
+[https://www.build.top/](https://www.build.top/)
+
+- BuildAI
+
+[https://buildaierc.com/](https://buildaierc.com/)
+
+## Tokenomics
+Tokenomics is one of the most important parts of a project evaluation. From the experts' point of view, the tokenomics of Ubiquity's tokens are opaque and risky. Although the distribution of UBQ tokens is unclear, presumably high inflation of UBQ tokens could lead to a decrease in the value of investments. Investors need to know the main parameters - token distribution, conditions (cliff, vesting, etc), and expected valuation. 
+
+Questions arise regarding the Ubiquity Dollar about mechanisms for maintaining stability, issuance, prevalence, and regulation. Most experts noted that most tokens launched in 2024 are losing their value, and they would like to see a plan for investor exits or for maintaining the value of UBQ.
+
+## Team
+Not counting Sergey, the Ubiquity team is anonymous, which is common for crypto projects. This does not significantly add or reduce investor trust. From the perspective of public promotion, it is necessary to have influencers who will be the voice of the project. Typically, investors want to see a cohesive team because companies with solo entrepreneurs are less stable.
```

Comment:
I think I will change my stance soon. This is because of the new regulatory environment (more pro crypto) in the US.